DataFormats/EgammaCandidates/interface/AODHIPhoton.h

```diff
@@ -0,0 +1,72 @@
+#ifndef AODHIPhoton_h
+#define AODHIPhoton_h
+
+#include "DataFormats/EgammaCandidates/interface/Photon.h"
+
+namespace aod{
+
+  class AODHIPhoton : public reco::Photon {
```

Comment:
see main thread about suggested reorganization to make a class containing just the new variables